manifests/charts/istio-control/istio-discovery/files/gen-istio.yaml

```diff
@@ -53,7 +53,7 @@ data:
         zipkin:
           address: zipkin.istio-system:9411
     enablePrometheusMerge: true
-    rootNamespace: null
+    rootNamespace: istio-system
```

Comment:
hmmm, is this really necessary? I thought that the code will default to istio-system if there is no specified.